src/index.js

```diff
@@ -0,0 +1,142 @@
+let currentTemperature = 0;
+
+const tempValue = document.getElementById('tempValue');
+
+const temperatureDisplay = () => {
```

Comment:
Typically function names have a verb so you know that they _do_. Consider naming this function something like `updateTemperatureDisplay`